src/analysis/Environments/Environments.ts

```diff
@@ -803,8 +803,9 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
         ]),
       errorRuntimeId &&
         h(RuntimeErrorModal, {
-          runtime: _.find({ id: errorRuntimeId }, runtimes),
+          runtime: _.find({ id: errorRuntimeId }, runtimes) as RuntimeWithWorkspace,
```

Comment:
Can we get rid of the cast if we use native JS here?

```suggestion
          runtime: runtimes.find(runtime => runtime.id === errorRuntimeId),
```

Reply:
the cast was more to not deal with the case where find() came up with no result (it should "never" happen since we are looking for it by ID that "should" be there cuz the list was the source of truth on the IDs to start with).... switching to useModalHandler can remove this dance since it supports the runtime object being the "arg" for the modal... but I held off on bringing that in.

Reply:
Ah. In that case, suggest a non-null assertion:

```suggestion
          runtime: _.find({ id: errorRuntimeId }, runtimes)!,
```